src/aten/ReduceOps.cpp

```diff
@@ -468,14 +469,70 @@ Tensor& XPUNativeFunctions::argmax_out(
     c10::optional<int64_t> dim,
     bool keepdim,
     Tensor& out) {
-  std::optional<Device> common_device = std::nullopt;
-  c10::impl::check_and_update_common_device(
-      common_device, out, "XPUNativeFunctions::argmax_out", "out");
-  c10::impl::check_and_update_common_device(
-      common_device, self, "XPUNativeFunctions::argmax_out", "self");
   out = argmax_meta(self, dim, keepdim, out);
   argmax_argmin_impl(self, dim, keepdim, out, native::xpu::argmax_kernel);
   return out;
 }
 
+Tensor XPUNativeFunctions::argmax(
+    const Tensor& self,
+    c10::optional<int64_t> dim,
+    bool keepdim) {
+  Tensor out;
+  out = argmax_meta(self, dim, keepdim, out);
+  argmax_argmin_impl(self, dim, keepdim, out, native::xpu::argmax_kernel);
+  return out;
+}
+
+void amax_impl(
+    Tensor& result,
+    const Tensor& input,
+    IntArrayRef dim,
+    bool keepdim) {
+  auto iter = native::make_reduction(
+      "amax", result, input, dim, keepdim, input.scalar_type());
+  native::xpu::max_all_launch_kernel(iter);
```

Comment:
Can you help to remove `launch` in name for all kernels in `src/aten/sycl/ReduceMaxValuesKernel.h`?